### PR TITLE
feat: adding dashboard extension banner

### LINKED ISSUE #6215: Include a widget for recommended extensions at the bottom of the dashboard page.
### Is your enhancement related to a problem? Please describe

subpart of https://github.com/containers/podman-desktop/issues/5771

Featured extension widget is nice but you just have short title/name and it's hard to know what it does

### Describe the solution you'd like

Add a banner to promote given extensions

it should be like featured extension but full-width widget with one extension at a time

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Edit (@axel7083)

- [x] #6656
- [x] https://github.com/containers/podman-desktop/issues/6669 
- [x] https://github.com/containers/podman-desktop/pull/6721
- [x] #6729

### PR DESCRIPTION
**Pull request by axel7083**
### What does this PR do?

This PR is the follow up of https://github.com/containers/podman-desktop/pull/6681, adding the UI components to display the recommended extension banner.

### Screenshot / video of UI

image

### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop/issues/6215

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

**manually**

Open the Dashboard and see the at the bottom the `RedHat extension pack`

- You can try installing it, the banner will disappear 
- If you uninstall it, it will re-appear

### CODE REVIEW

**Review by benoitf — APPROVED** [review]
codewise LGTM but wait for @ekidneyrh review or 24h before merging

**Review by ekidneyrh — APPROVED** [review]
LGTM :)

**Comment by ekidneyrh** [comment]
Hi Axel! Apologies for going back on my LGTM!! @benoitf pointed out the similarity in the bg of the dashboard page and on the banner. Would it be possible to change the bg svg to this one:

Red Hat extension banner

 And the extension area to charcoal-600?

**Comment by axel7083** [comment]
image

Here is the rebased/updated version following https://github.com/containers/podman-desktop/pull/6721

cc @ekidneyrh @benoitf

**Review by benoitf — APPROVED** [review]
we'll need to allow to provide optional light/dark images in recommendations.json for supporting light theme but it's a follow-up

**Comment by slemeur** [comment]
Can we have the banner at the top of the featured extension? (before the list of the 6 featured exts)

**Comment by axel7083** [comment]
> Can we have the banner at the top of the featured extension? (before the list of the 6 featured exts)

Yes, here is the new screenshot

image

**Comment by benoitf** [comment]
AFAIK It should be at the bottom for now (as mentioned by the issue) (bottom is not the second from the bottom)

As Featured Extension widget will be removed and be visible in the Extension catalog page it'll be moved up automatically at that time

and it's to avoid "visual duplicates" on the first show
image

**Comment by benoitf** [comment]
it's missing the EXTENSION text from the mockup appearing on top of the icon

image

File: packages/renderer/src/lib/dashboard/DashboardPage.svelte
Comment (axel7083) [inline packages/renderer/src/lib/dashboard/DashboardPage.svelte]:
```suggestion
        <FeaturedExtensions />
        <ExtensionBanners />
```

**Comment by axel7083** [comment]
Following the merge of https://github.com/containers/podman-desktop/pull/6730 here is a post updated/rebased screenshot

image

**Review by benoitf — APPROVED** [review]

**Comment by slemeur** [comment]
Just to confirm.

In the feature extension section, we'll have only the banners? 
And a more prominent button to access the catalog of extension?

(trying to clarify my understand of this [comment](https://github.com/containers/podman-desktop/pull/6708#issuecomment-2045117821))

**Pull request merged**
